src/QsCompiler/Core/SymbolResolution.fs

```diff
@@ -553,8 +562,9 @@ module SymbolResolution =
     /// and generates suitable diagnostics if they are not, replacing them by the Q# type denoting an invalid type.
     /// Returns the resolved type as well as an array with diagnostics.
     /// IMPORTANT: for performance reasons does *not* verify if the given the given parent and/or source file is inconsistent with the defined callables.
-    /// May throw an ArgumentException if no namespace with the given name exists, or the given source file is not listed as source of that namespace.
-    /// Throws a NotSupportedException if the QsType to resolve contains a MissingType.
+    /// </summary>
+    /// <exception cref="NotSupportedException"><paramref name="qsType"/> contains a <see cref="MissingType"/>.</exception>
+    /// <exception cref="ArgumentException">No namespace with the given name exists, or the given source file is not listed as source of that namespace.</exception>
```

Comment:
I can't remember where exactly our discussion on this landed, but did we want to split these kinds of unrelated conditions into two tags?

```suggestion
    /// <exception cref="ArgumentException">No namespace with the given name exists</exception>
    /// <exception cref="ArgumentException">The given source file is not listed as source of that namespace.</exception>
```

Reply:
I think we decided to split them up for completely unrelated causes (like the case we have here), since it's then easy to generate documentation with a formatting that makes the separate causes easy to distinguish. LGTM!